src/modules/favorites/types.ts

```diff
@@ -22,7 +22,7 @@ export type SearchLocation = {
   category: FeatureCategory[];
   label?: string;
   postalcode?: string;
-  tariff_zones?: string[];
+  fare_zones?: string[];
```

Comment:
I don't think we can or should change this field. It's connected to the type returned by the geocoder, which I guess haven't changed?

(Also, we store data on this format in local storage, so when changing it, we need to consider backwards compatibility)